docs/patterns/observability/README.md

```diff
@@ -0,0 +1,7 @@
+# Observability
+
+OSM's observability stack includes Prometheus for metrics collection, Grafana for metrics visualization and Fluent Bit for log forwarding to a user-defined endpoint. These are disabled by default but may be enabled during OSM installation with flags `--enable-prometheus`, `--enable-grafana` and `--enable-fluentbit`. Alternatively, their values may be set in `.env`.
```

Comment:
> Alternatively, their values may be set in `.env`.

Is this still for install? not seeing env values grabbed at install time on install.go (cli).

Reply:
oh you're right, I think I got confused and that's true only for the demo. Removing now!